index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <a href="">Projects</a>
+    </nav>
+    <section id="welcome-section">
+      <h1>Hey we are Software Developers</h1>
+    </section>
+
+    <section id="projects">
+      <h2>These are some of our Projects</h2>
+      <div class="project-tile">
+        <img src="tribute.jpg" alt="tribute page">
+      </div>
+      <a href=""></a>
+    </section>
+    <a id="profile-link" href="">My GitHub</a>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as a comment makes your ode not clean